<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyProxy</class>
 <widget class="QWidget" name="MyProxy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>类型</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="Type">
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>HTTP</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Socks5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Caching</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>代理  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="HostName"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>用户：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="User"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>端口  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Port"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>密码：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Password"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="BtnOK">
       <property name="text">
        <string>确定</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>Type</tabstop>
  <tabstop>HostName</tabstop>
  <tabstop>Port</tabstop>
  <tabstop>User</tabstop>
  <tabstop>Password</tabstop>
  <tabstop>BtnOK</tabstop>
  <tabstop>BtnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
